db/migrate/20220922091238_create_comments.rb

```diff
@@ -1,9 +1,17 @@
 class CreateComments < ActiveRecord::Migration[7.0]
   def change
     create_table :comments do |t|
+      t.integer :author_id, foreign_key: true
+      t.integer :post_id, foreign_key: true
       t.text :text
 
       t.timestamps
     end
+    add_foreign_key :comments, :users, column: :author_id
+    add_index :comments, :author_id
```

Comment:
- [ ] I am getting an error when trying to run your migrations, kindly take a step back and check the migrations
![image](https://user-images.githubusercontent.com/62705947/191984006-110b7849-5a3c-41df-9cf6-85fc832c182e.png)